test/multicluster/minikube_base_multicluster_test.go

```diff
@@ -74,6 +74,9 @@ func TestKubernetesBasicMultiCluster(t *testing.T) {
 			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
 			"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
 			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			// K8s version of AKS cluster does not support generic ephemeral volumes
+			"admin.ephemeralVolume.enabled":    "false",
+			"database.ephemeralVolume.enabled": "false",
```

Comment:
Why do we need to explicitly disable the ephemeral volumes when they should be disabled by default?

Reply:
I'll remove this. I added this when I temporarily set the default to true to enable testing without having to create a new HelmFeature.